code/modules/admin/admin_verbs.dm

```diff
@@ -597,4 +597,39 @@ ADMIN_VERB_ADD(/client/proc/toggleUIDebugMode, R_DEBUG, FALSE)
 	if(UI)
 		UI.toggleDebugMode()
 	else
-		log_debug("This mob has no UI.")
\ No newline at end of file
+		log_debug("This mob has no UI.")
+
+ADMIN_VERB_ADD(/client/proc/create_portals, R_ADMIN, FALSE)
+/client/proc/create_portals()
+	set category = "Fun"
+	set name = "Create Portals"
+	set desc = "Create bi-directional portals between two locations."
+
+	var/x_1 = input(src, "X Coordinate", "First Portal Location") as null|num
+	if(!x_1)
+		return
+	var/y_1 = input(src, "Y Coordinate", "First Portal Location") as null|num
+	if(!y_1)
+		return
+	var/z_1 = input(src, "Z Coordinate", "First Portal Location") as null|num
+	if(!z_1)
+		return
+
+	var/x_2 = input(src, "X Coordinate", "Second Portal Location") as null|num
+	if(!x_2)
+		return
+	var/y_2 = input(src, "Y Coordinate", "Second Portal Location") as null|num
+	if(!y_2)
+		return
+	var/z_2 = input(src, "Z Coordinate", "Second Portal Location") as null|num
+	if(!z_2)
+		return
+
+	// Spawning junk tractor beam portals because they never disappear
```

Comment:
Not sure this is a good idea if JTB portals ever get changed.

Reply:
Alright I motivated myself and created a new subtype of portal to avoid future issues.